stdlib/io-console/0/io-console.rbs

```diff
@@ -0,0 +1,137 @@
+class IO
+  class ConsoleMode
+    def echo=: ((true | false)) -> (true | false)
+    def raw: (?min: int, ?time: int, ?intr: (true | false)) -> self
+    def raw!: (?min: int, ?time: int, ?intr: (true | false)) -> self
+  end
+
+  # Returns an File instance opened console.
+  #
+  # If `sym` is given, it will be sent to the opened console with `args` and the
+  # result will be returned instead of the console IO itself.
+  def self.console: () -> File?
+                  | (:close) -> nil
+                  | (Symbol sym, *untyped args) -> top
+
+  # returns console window size
+  #
+  # You must require 'io/console/size' to use this method.
+  def self.console_size: () -> [Integer, Integer]
+
+  # fallback to console window size
+  #
+  # You must require 'io/console/size' to use this method.
+  def self.default_console_size: -> [Integer, Integer]
+
+  def beep: () -> self
+
+  def check_winsize_changed: () { () -> void } -> self
+  def clear_screen: () -> self
+
+  # Returns a data represents the current console mode.
+  def console_mode: () -> IO::ConsoleMode
+
+  # Sets the console mode to `mode`.
+  def console_mode=: (IO::ConsoleMode mode) -> IO::ConsoleMode
+
+  # Yields `self` within cooked mode.
+  #
+  #     STDIN.cooked(&:gets)
+  #
+  # will read and return a line with echo back and line editing.
+  def cooked: () { (self) -> void } -> void
```

Comment:
```suggestion
  def cooked: [T] () { (self) -> T } -> T
```

Because `IO#cooked` returns the block's returned value.

```ruby
STDIN.cooked {42} # => 42
```

Reply:
I didn't know how to write here. Thank you ✨